app/code/Transoft/Blog/Api/ModelRepositoryInterface.php

```diff
@@ -0,0 +1,48 @@
+<?php
```

Comment:
Please specify throwed exceptions for each method. Now they are incompatible with implementation

Reply:
fixed, but not sure if it should be that way